setup.py

```diff
@@ -7,9 +7,12 @@
 )
 
 tests_require = (
-    'nose'
+    'nose',
 )
 
+setup_requires = (
+    'flake8',
```

Comment:
I don't think your setup should require `flake8`. It's a linter tool and not a runtime dependency for your code. If you want to test with it, it should be in `tests_require`.